services/explorer/src/pages/Transactions/Transactions.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Table } from '@taraxa_project/taraxa-ui';
+import { Container, Typography } from '@mui/material';
+import { toTableRow } from '../../components/TransactionRow/TransactionRow';
+import { useTransactionsEffects } from './Transactions.effect';
+
+const TransactionsPage = () => {
+  const { data, columns } = useTransactionsEffects();
+  const rows = data ? data.map((row) => toTableRow(row)) : [];
+  return (
+    <Container maxWidth='xl'>
```

Comment:
I think we will need this everywhere including the header and footer, right?
Can't we add it in the App.tsx as a wrapper for the whole application, which includes, header and footer?